assets/responsibleai/environments/responsibleai-vision-ubuntu20.04-py38-cpu/context/Dockerfile

```diff
@@ -33,9 +33,10 @@ RUN pip install 'shap==0.40.0' \
                 'setuptools>=65.5.1' \
                 'numpy==1.22.0' \
                 'scipy==1.10.0' \
-                'statsmodels==0.14.0'
+                'statsmodels==0.14.0' \
+                'torch==1.13.1'

```

Comment:
this needs to be fixed in automl and not in our image - also, the fix is more complex than this, since we need to install it in conda

Reply:
we've had success pinning statsmodel to a higher version by pinning the package in docker after conda installation is done. This pin for statsmodel fixed vulnerability due to automl requiring a lower version.

Reply:
yes and I agree this needs to be fixed in automl. We created a thread in automl channel, I think we should escalate there as well but before then we should also fix it here since this has a due date of 09/01.